docs/mara_lampert/getting_started_with_mambaforge_and_python/readme.md

```diff
@@ -0,0 +1,199 @@
+# Getting started with Mambaforge and Python 
+[Mara Lampert](../readme), January 26th, 2023
+
+## Introduction to Python and Mamba 
+This blog post explains what Python and [Mamba](https://mamba.readthedocs.io/en/latest/installation.html)/ Mambaforge, and how you can download and setup it on your computer. We will also go through some steps how to get started with Bio-image Analysis. 
+
+> **_Note:_** This is an update of a [previous Blogpost](https://biapol.github.io/blog/johannes_mueller/anaconda_getting_started/) written by Johannes. 
+
+
+**Why do we need Mamba to use Python?**
+
+__Python__ is a programming language which is easy to learn and very important in scientific data analysis nowadays. 
+
+__Mamba__ is a __package manager__ which can be used with Python. It is a software allowing to install other software. Read more about Mamba [here](https://focalplane.biologists.com/2022/12/08/managing-scientific-python-environments-using-conda-mamba-and-friends/). 
+
+## Installation of Mambaforge 
+Here, I am going to show how to install Mambaforge. It comes with everything you need and downloads it from a community-driven open source software provider called  [conda-forge](https://conda-forge.org/). 
+First, you pick the Mambaforge installer for your operating system [here](https://github.com/conda-forge/miniforge#mambaforge):
+
+![](imgs/1_mambaforge_download_1.png)
+
+All Mac OS users can now jump [here](#installation-on-mac-os).
+
+### Installation on Windows
```

Comment:
When installing mambaforge, I get a prompt by windows that looks like this:
![Screenshot 2023-01-27 081340](https://user-images.githubusercontent.com/38459088/215033863-125545ce-e57f-4f35-8f57-cbb23af22779.jpg)

I have to click "Trust Publisher" ("Trotzdem ausführen" in German). I'm not sure how this would look like in english, passing this window also doesn't require admin privileges. Maybe mention it anyway :)

Reply:
Check first if this happens on all computers. Keep the instructions simple